static/src/javascripts/projects/common/modules/commercial/adblock-banner-config.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { banners } from './adblock-banners';
+
+const getBanners = function(edition: string): Array<any> {
```

Comment:
Yeah, lots of JSON manipulation going on that makes it complicated to define exactly 😕